File: backend/core/src/main/java/site/timecapsulearchive/core/domain/capsuleskin/data/response/CapsuleSkinsPageResponse.java
package site.timecapsulearchive.core.domain.capsuleskin.data.response;

import io.swagger.v3.oas.annotations.media.Schema;
import java.util.List;

@Schema(description = "캡슐 스킨 페이징")
public record CapsuleSkinsPageResponse(

    @Schema(description = "캡슐 스킨 요약 정보 리스트")
    List<CapsuleSkinSummaryResponse> skins,

    @Schema(description = "다음 페이지 유무")
    Boolean hasNext
) {

}

File: frontend/ARchive/app/src/main/java/com/droidblossom/archive/domain/model/capsule_skin/CapsuleSkinsPageResponse.kt
package com.droidblossom.archive.domain.model.capsule_skin

import com.droidblossom.archive.domain.model.common.CapsuleSkinSummary

data class CapsuleSkinsPageResponse(
    val skins : List<CapsuleSkinSummary>,
    val hasNext : Boolean
)
